server/api/controllers/volunteer-application-controller.ts

```diff
@@ -91,7 +96,7 @@ export class VolunteerApplicationController {
         "reply-to": formBody.address,
       },
       subject: "Volunteer Application - " + formBody.role,
-      text: `From: ${formBody.firstName}${formBody.preferredName ? ` "${formBody.preferredName}" ` : " "}${formBody.lastName} <${formBody.address}>\n
+      text: `From: ${formBody.firstName}${formBody.preferredName ? ` (${formBody.preferredName}) ` : " "}${formBody.lastName} <${formBody.address}>\n

```

Comment:
Thoughts on if we even need the first name if we have a preferred name? Since this isn't a legal application

Reply:
Oh yeah we 100% don't need it, good catch!